examples/vg-specs/bar_grouped.vg.json

```diff
@@ -264,20 +264,11 @@
                     },
                     "encode": {
                         "update": {
-                            "width": {
-                                "signal": "child_width"
-                            },
-                            "height": {
-                                "signal": "child_height"
-                            },
                             "stroke": {
                                 "value": "#ccc"
                             },
                             "strokeWidth": {
                                 "value": 0
-                            },
-                            "fill": {
```

Comment:
My vote would be to keep the transparent fills, unless there's a compelling reason to get rid of them. Having an undefined fill is a frequent source of bugs with capturing interaction events.